webapp/src/container.tsx

```diff
@@ -622,27 +622,30 @@ export class SideDocs extends data.Component<SideDocsProps, SideDocsState> {
         this.popOut = this.popOut.bind(this);
     }
 
+    private rootDocsUrl(): string {
+        return pxt.webConfig.docsUrl || '/--docs#';
```

Comment:
Isn't pxt.webConfig.docsUrl sent from backend without the #? (As just `/--docs`)

Reply:
great catch